documentation/2.0/content/release-notes/release-2.4.1.md

```diff
@@ -0,0 +1,28 @@
++++
+title = "Release 2.4.1"
+date = 2022-11-03T15:27:38-05:00
+weight = 5
+pre = "<b> </b>"
++++
+
+### Changes in Release 2.4.1
+- [Major New Features](#major-new-features)
+- [Other Changes](#other-changes)
+- [Bugs Fixes](#bug-fixes)
+- [Known Issues](#known-issues)
+
+
+#### Major New Features
+- None
+
+#### Other Changes
+None
+
+#### Bug Fixes
+- #1237: Resolved an issue where the tools supporting the `-target` parameter were doing target-related work even if the parameter was not specified.
```

Comment:
even if the parameter was not specified -> even when the parameter was not specified